Description: enable multiarch installation paths
Author: Ghislain Antony Vaillant <ghisvail@gmail.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- arrayfire.orig/CMakeLists.txt
+++ arrayfire/CMakeLists.txt
@@ -69,7 +69,7 @@
 IF(WIN32)
     SET(ARRAYFIRE_CMAKE_DIR cmake)
 ELSE()
-    SET(ARRAYFIRE_CMAKE_DIR share/ArrayFire)
+    SET(ARRAYFIRE_CMAKE_DIR lib/${CMAKE_LIBRARY_ARCHITECTURE}/cmake/ArrayFire)
 ENDIF()
 
 IF(${BUILD_CPU})
--- arrayfire.orig/src/backend/cpu/CMakeLists.txt
+++ arrayfire/src/backend/cpu/CMakeLists.txt
@@ -83,7 +83,7 @@
     VERSION "${AF_VERSION}"
     SOVERSION "${AF_VERSION_MAJOR}")
 
-INSTALL(TARGETS afcpu EXPORT CPU DESTINATION lib COMPONENT libraries)
+INSTALL(TARGETS afcpu EXPORT CPU DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE} COMPONENT libraries)
 
 export(TARGETS afcpu FILE ArrayFireCPU.cmake)
 INSTALL(EXPORT CPU DESTINATION ${ARRAYFIRE_CMAKE_DIR} 
